packages/edit-post/src/components/layout/index.js

```diff
@@ -116,7 +117,8 @@ function Layout( { styles } ) {
 			showIconLabels:
 				select( editPostStore ).isFeatureActive( 'showIconLabels' ),
 			isDistractionFree:
-				select( editPostStore ).isFeatureActive( 'distractionFree' ),
+				select( editPostStore ).isFeatureActive( 'distractionFree' ) &&
+				isLargeViewport,
```

Comment:
Should `isLargeViewport` be a dependency of the `useSelect` in order that it will re-run when the viewport changes?

Reply:
same here